values.yaml

```diff
@@ -55,7 +55,7 @@ commonAnnotations: {}
 image:
```

Comment:
Pouet!

the bump for the upgrade image is missing in this file (path to the variable: `upgrade.image.tag`)

Reply:
Added